### PR TITLE
#131157821 Bug - logo on community creation

### PR DESCRIPTION
**Pull request by andela-jkamau**
# What does this PR do?

It fixes a bug where images were not being uploaded to Cloudinary when creating a community
# Description of Task to be completed?
1. Make sure that logo is saved. Add tests for right dimension and the right size. Add tests for saving logo( Cloudinary mocking.)
2. Enforce size and dimension: When adding a logo during community creation please check that put a check that enforces a size check less than 36KB.  And also the 100 X 100 dimension.
# How should this be manually tested?
- Create an account and login with your username and password
- Click on the circled '+' button next to the COMMUNITIES  header
- Add the details as required on the **Create A New Community** form and choose an image. Submit the form.
- Click on 'View All' to see the community you've created
# What are the relevant pivotal tracker stories?
# 131157821 - Fix bugs with logo on community creation
# Screenshots

screen shot 2016-10-25 at 4 10 55 pm

### CODE REVIEW

File: codango/community/tests/test_community.py [hunk 1]
```diff
@@ -24,14 +26,20 @@ def create_community(self, private=False, visibility='full'):
 
     def test_community_create(self):
         """Test that users can create community"""
+        # image_abs_path = os.path.abspath('static/img/sample.png')
```
Comment (andela-cobiora) [inline codango/community/tests/test_community.py]: Delete commented out line

File: codango/community/tests/test_community.py [hunk 2]
```diff
@@ -46,6 +54,18 @@ def test_community_create(self):
...
                                                        community=Community.objects.get(name='Test Community')))
 
+        # Test logo is being uploaded
+        self.assertIn(
+            'sample', Community.objects.filter(
+                name='Test Community')[0].logo.public_id)
+
+        # Delete the logo from Cloudinary's CDN
+        cloudinary.uploader.destroy(
+            str(Community.objects.filter(
```
Comment (andela-cobiora) [inline codango/community/tests/test_community.py]: Indentation is off here, refer to PEP 8 style guide

File: codango/community/tests/test_community.py [hunk 3]
```diff
@@ -24,14 +26,20 @@ def create_community(self, private=False, visibility='full'):
 
     def test_community_create(self):
         """Test that users can create community"""
+        # image_abs_path = os.path.abspath('static/img/sample.png')
+        image_abs_path = \
+            '/home/ubuntu/codango/codango/static/img/sample.png'
```
Comment (andela-cobiora) [inline codango/community/tests/test_community.py]: Fix up to not use hardcoded path string

File: codango/community/tests/test_community.py [hunk 4]
```diff
@@ -46,6 +54,18 @@ def test_community_create(self):
         self.assertTrue(CommunityMember.objects.filter(user=self.user,
                                                        community=Community.objects.get(name='Test Community')))
 
+        # Test logo is being uploaded
+        self.assertIn(
+            'sample', Community.objects.filter(
```
Comment (andela-cobiora) [inline codango/community/tests/test_community.py]: Indentation is off here, refer to PEP 8 style guide

**Comment by andela-cobiora** [comment]
Squash the last two commits using git rebase

File: codango/community/tests/test_community.py (hunk 4 above)
Comment (andela-jkamau) [inline codango/community/tests/test_community.py]: I'm getting PEP8 errors if I change the indentation

**Pull request merged**